core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -227,7 +228,8 @@ object TestUtils extends Logging {
     val props = new Properties
     if (nodeId >= 0) props.put(KafkaConfig.BrokerIdProp, nodeId.toString)
     props.put(KafkaConfig.ListenersProp, listeners)
-    props.put(KafkaConfig.LogDirProp, TestUtils.tempDir().getAbsolutePath)
+    val logDir = (1 to logDirCount).toList.map(i => TestUtils.tempDir().getAbsolutePath).mkString(",")
+    props.put(KafkaConfig.LogDirProp, logDir)
```

Comment:
I updated the code to use `KafkaConfig.LogDirsProp` only if the dir count > 1. This is because some tests such as OffsetCommitTest.scala assumes that there is only one log directory and the will fetch the log directory based on `log.dir`. It seems simpler to keep these tests and only update the `TestUtils.createBrokerConfig()` to fill in the property based on the directory number.

```
    if (logDirCount > 1) {
      val logDirs = (1 to logDirCount).toList.map(i => TestUtils.tempDir().getAbsolutePath).mkString(",")
      props.put(KafkaConfig.LogDirsProp, logDirs)
    } else {
      props.put(KafkaConfig.LogDirProp, TestUtils.tempDir().getAbsolutePath)
    }
```